src/components/expression/expression.js

```diff
@@ -172,6 +204,22 @@ genotet.ExpressionView = function(viewName, params) {
     .on('genotet.updatePanel', function() {
       this.panel.dataLoaded();
     }.bind(this));
+
+  // Set up link callbacks.
+  $(this).on('genotet.addProfile', function(event, data) {
+    var genes = data.id.split(',');
```

Comment:
Here for some reasons you need to retrieve the ids from a comma separated string data.id. You can replace the comma separated string with !Array<string> and I think it will work too.